atlas/schema.hcl

```
@@ -1,11 +1,476 @@
+      increment = 1
+    }
+  }
+  column "title" {
+    null = false
+    type = text
+    comment = "name of business"
+  }
+  column "income_type" {
+    null = false
+    type = text
+  }
+  column "industry_code" {
+    null = false
+    type = text
+  }
+  column "main_product" {
+    null = true
+    type = text
+    comment = "main product/service sold by the business"
+  }
+  column "partnership" {
+    null = false
+    type = bool
+    default = false
+  }
+  column "address" {
+    null = true
+    type = text
+    comment = "business physical address if applicable"
+  }
+  column "create_time" {
+    null = false
+    type = time
+  }
+  primary_key {
+    columns = [column.organization_id, column.snapshot_id]
+  }
+  foreign_key "organization_id" {
+    columns     = [column.organization_id]
+    ref_columns = [table.organization_v1.column.organization_id]
+    on_update   = CASCADE
+    on_delete   = CASCADE
+  }
+}
+table "organization_v1" {
+  schema = schema.public
+  comment = "Refer to the link for explanations of the organization table fields: https://help.wealthsimple.com/hc/en-ca/articles/4408339655323-How-do-I-report-my-self-employment-income-on-a-T2125"
+  column "organization_id" {
+    null = false
+    type = bigint
+    identity {
+      generated = ALWAYS
+      start = 0
+      increment = 1
+    }
+  }
+ column "title" {
+    null = false
+    type = text
+    comment = "name of business"
+  }
+  column "income_type" {
+    null = false
+    type = text
+  }
+  column "industry_code" {
+    null = false
+    type = text
+  }
+  column "main_product" {
+    null = true
+    type = text
+    comment = "main product/service sold by the business"
+  }
+  column "partnership" {
+    null = false
+    type = bool
+    default = false
+  }
+  column "address" {
+    null = true
+    type = text
+    comment = "business physical address if applicable"
+  }
+  column "create_time" {
+    null = false
+    type = time
+  }
+  primary_key {
+    columns = [column.organization_id]
+  }
+}
+table "project_v1" {
+  schema = schema.public
+  column "user_id" {
+    null = false
+    type = bigint
+  }
+  column "project_id" {
+    null = false
+    type = bigint
+    identity {
+      generated = ALWAYS
+      start = 0
+      increment = 1
+    }
+  }
+  primary_key {
+    columns = [column.user_id, column.project_id]
+  }
+  foreign_key "user_id" {
+    columns     = [column.user_id]
+    ref_columns = [table.user_v1.column.user_id]
+    on_update   = CASCADE
+    on_delete   = CASCADE
+  }
+  index "project_id_idx" {
+    unique  = true
+    columns = [column.project_id]
+  }
+}
+table "scan_inference_v1" {
+  schema = schema.public
+  column "scan_id" {
+    null = false
+    type = bigint
+  }
+  column "scan_inference_id" {
+    null = false
+    type = bigint
+    identity {
+      generated = ALWAYS
+      start = 0
+      increment = 1
+    }
+  }
+  column "inference_result" {
+    null = false
+    type = bytea
```

pretty sure there is a more specific datatype for json
```suggestion
    type = jsonb
```

---

I read that bytea is more performance oriented so my thought process here was convert back to JSON in the backend

---

no jsonb is the correct type here

overindexing on performance will lead to much pain down the road